### PR TITLE
payment link fix  and blog url fix

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/router/routes.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const routes = [
     // which is lazy-loaded when the route is visited.
     component: () => import('@/views/Faq.vue'),
     meta: {
-      title: "FAQ's - Get your all queries be answered..."
+      title: "FAQ's - Get Your All Queries be Answered..."
```
Comment (arrogant-ant) [inline src/router/routes.js:23]: add some keywords like parking
"Parking related queries"

File: src/pages/faq/components/PSFaq.vue [hunk 2]
```diff
@@ -72,6 +72,26 @@
...
       </article>
+
+      <article class="message">
+        <div class="message-header">
+          <p>How can I rent my empty parking space?</p>
+        </div>
+        <div class="message-body">
+          <p>Follow below simple steps:-</p>
+          <p>
+            1) Contact us directly via phone (+91 80929 96057).We will take some
```
Comment (arrogant-ant) [inline src/pages/faq/components/PSFaq.vue:83]: Please add space after full stop ```) .We will```

File: src/pages/faq/components/PSFaq.vue [hunk 3]
```diff
@@ -72,6 +72,26 @@
...
+            details from you and your empty space is get listed on our platform.
+          </p>
+          <p>
+            2) Email us at
+            <a href="mailto: connect@parkspot.in" style="color: black">
+              connect@parkspot.in</a
+            >
+            with the request.We will contact you and process the request.
```
Comment (arrogant-ant) [inline src/pages/faq/components/PSFaq.vue:91]: here too

File: src/views/payment-gateway.vue [hunk 4]
```diff
@@ -31,49 +31,60 @@ export default {
...
-        name: data.BookingInfo.Name,
-        dueDate: data.DueDate,
-        amount: data.Fee.Amount,
-        discount: data.Fee.Discount,
-        convenienceFee: data.Fee.ConvenienceFee,
-      };
-      this.paymentMode = { ...data.Payment };
-      // console.log(this.bookingDetails);
+      } catch (exception) {
+        // console.log("Error");
```
Comment (arrogant-ant) [inline src/views/payment-gateway.vue:78]: if not needed can we delete all commented console log

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/router/routes.js (hunk 1 above)
Comment (alias01) [reply src/router/routes.js:23]: done

File: src/pages/faq/components/PSFaq.vue (hunk 3 above)
Comment (alias01) [reply src/pages/faq/components/PSFaq.vue:91]: done

**Pull request merged**